SwiftCheck/Test.swift

```diff
@@ -223,8 +223,10 @@ internal func quickCheckWithResult(args : Arguments, p : Testable) -> Result {
 					, randomSeed:			rnd()
 					, numSuccessShrinks:	0
 					, numTryShrinks:		0
-					, numTotTryShrinks:		0)
-	return test(state, p.property().unProperty.unGen)
+					, numTotTryShrinks:		0
+					, shouldAbort:			false)
+	let modP : Property = (p.exhaustive ? once(p.property()) : p.property())
```

Comment:
The damned unifier wouldn't let me write this as 

```swift
test(state, (p.exhaustive ? once : id)(p.property()).unProperty.unGen)
```

because it wouldn't instantiate the `Testable` constraint.  Will radar.